index.html

```diff
@@ -146,8 +146,8 @@ <h2>Extensions to the <dfn data-cite="HTML5#the-window-object">Window</dfn> inte
     </section>
     <section>
       <h2><dfn data-lt="ContinuousEvent">ContinuousEvents</dfn></h2>
-      <p>A task is considered to be a <code>ContinuousEvent</code> if the task will dispatch an event that is a <a>UIEvent</a> or <a>TouchEvent</a> or a child of either and if for events of this type the user agent typically fires multiple events of this type in succession at a rate that is implementation and platform specific.</p>
-      <p class="note">Generally the events that are considered to be continous are <dfn data-cite="uievents#event-type-mousemove">mousemove</dfn>, <dfn data-cite="uievents#event-type-wheel">wheel</dfn> and <dfn data-cite="touch-events#the-touchmove-event">touchmove</dfn> all of which are specified in such a way where the browser is encouraged to fire the events at a rate specific for the browser and platform. Events that are introduced in the future and similar should also probably be considered to be continous.</p>
+      <p>A task MUST be considered a <code>ContinuousEvent</code> if the task will dispatch an event that is a <dfn data-cite="uievents#event-type-mousemove">mousemove</dfn>, <dfn data-cite="uievents#event-type-wheel">wheel</dfn>, <dfn data-cite="touch-events#the-touchmove-event">touchmove</dfn>, <dfn data-cite="html#dragevent">drag</dfn>, or <dfn data-cite="pointerevents#the-pointermove-event">pointermove</dfn> or a child of any of those. A user agent MAY consider other tasks that will dispatch events that are of a specific type and are children of <a>UIEvent</a> or <a>TouchEvent</a> to be a <code>ContinuousEvent</code> as long as those specific types are consistent for that user agent and that all tasks that will dispatch events of those types or children of those types are considered to be <code>ContinuousEvents</code>.</p>
```

Comment:
Can we rename it ContinuousEventTask? And define it in terms of ContinuousEvent, which would be a (trusted) event of the types mentioned?

Reply:
Yep, that makes sense.